packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -191,8 +204,19 @@ class ToCiceroMarkVisitor {
                             ...nodeInformation.properties,
                             `${NS_PREFIX_CommonMarkModel}Strong`,
                         ];
+                    } else if (runTimeProperties.name === 'w:color') {
+                        if (runTimeProperties.attributes['w:val'] === 'C7254E') {
+                            colorCodePresent = true;
+                        }
+                    } else if (runTimeProperties.name === 'w:shd') {
```

Comment:
Just comment here why you are using `shd` instead of hightlight.

Reply:
Can you give insights here?
According to me
> Limited highlighting in word so use shade for highlting.

Reply:
`w:shd` tag is used to detect the highlight colour of the text. `w:highlight` is not used for this purpose because of the fixed colours provides. Reference: http://officeopenxml.com/WPtextShading.php.